user account service test implemented

sakurano92 opened the pull request:
User account service test for few test that left out blank
has been implemented

sakurano92 committed:
user account service test update

simple test case is updated

dubeme commented on app/wama.core.unit.test/Services/UserAccountServiceTests.cs:
```diff
@@ -117,22 +122,40 @@ public UserAccountServiceTests()
-        [Fact]
-        public async Task SuspendUserAccountAsyncTest()
+        [Theory(DisplayName = "Approve UserAccounts")]
+        [InlineData("8521473", 1)]
```
Change the numbers to an actual boolean value (**_true_** or **_false_**)

sakurano92 replied:
oh yea forgot about it

dubeme reviewed (changes requested):
Not bad, the main issue has to do with you using numbers for boolean. For readability, and reducing level of confusion, switch to using an actual boolean value (true or false).

sakurano92 committed:
UserAccountService Test

change 0 and 1 to true and false

sakurano92 committed:
[BUGFIX]Report user set active not functional

Report user set active was not functional since only had code
for it with in poster not in get, so fixed.

iamcreasy commented:
expectedCount in GetPendingUserAccountsAsyncTest test is being testing against zero. Is this correct?

quazi-irfan commented on app/wama.core.unit.test/Services/UserAccountServiceTests.cs:
```diff
@@ -99,10 +99,15 @@ public UserAccountServiceTests()
+        [InlineData(UserAccountType.Employee, 0)]
+        [InlineData(UserAccountType.Manager, 0)]
+        [InlineData(UserAccountType.Patron, 0)]
+        public async Task GetPendingUserAccountsAsyncTest(UserAccountType accountType, int expectedCount)
```
expectedCount in GetPendingUserAccountsAsyncTest test is being testing against zero. Is this correct?
